apps/web/pages/signup.tsx

```diff
@@ -72,11 +75,12 @@ export default function Signup({ prepopulateFormValues, token, orgSlug }: Signup
       .then(async () => {
         telemetry.event(telemetryEventTypes.signup, collectPageParameters());
         const verifyOrGettingStarted = flags["email-verification"] ? "auth/verify-email" : "getting-started";
+
         await signIn<"credentials">("credentials", {
           ...data,
           callbackUrl: `${
             searchParams?.get("callbackUrl")
-              ? `${WEBAPP_URL}/${searchParams.get("callbackUrl")}`
+              ? `${searchParams.get("callbackUrl")}`
```

Comment:
This could be tampered to redirect you to unwanted sites isn't it? cc @hariombalhara

Reply:
should be fine now with getSafeCallbackUrl